skimage/data/_binary_blobs.py

```diff
@@ -30,7 +33,7 @@ def binary_blobs(length=512, blob_size_fraction=0.1, n_dim=2,
     Examples
     --------
     >>> from skimage import data
-    >>> data.binary_blobs(length=5, blob_size_fraction=0.2, seed=1)
+    >>> data.binary_blobs(length=5, blob_size_fraction=0.2)  # doctest: +SKIP
```

Comment:
Is this really the better way?

Reply:
In this case I could use a `seed` generated randomly and add `# do not copy` as done in other examples (this is not perfect but we agreed in previous comments that this was an OK compromise). I guess I forgot this one. I can do this if preferred.

Reply:
hmm ok. yeah i would rather people not copy the seed. I see this kind of copy from examples stuff too often.